runtime/src/bank.rs

```diff
@@ -9884,7 +9893,9 @@ pub(crate) mod tests {
 
         let (mut genesis_config, _mint_keypair) = create_genesis_config(1_000_000);
         for feature_id in FeatureSet::default().inactive {
-            if feature_id != solana_sdk::feature_set::set_exempt_rent_epoch_max::id() {
+            if feature_id != solana_sdk::feature_set::set_exempt_rent_epoch_max::id()
+                && feature_id != feature_set::skip_rent_rewrites::id()
```

Comment:
we should have tests that do have this feature activated, right?

Reply:
many tests activate all features. This specific one doesn't work with the feature activated, iirc.

Reply:
ha. turns out this test no longer fails anyway. I removed that.

Reply:
Ah, that's right 🤦‍♂️ . Thanks!